schemas/capacity-response.yml

```diff
@@ -0,0 +1,46 @@
+---
+  title: "WorkerType Capacity Response"
+  type: object
+  properties:
+    running:
+      type: object
+      properties:
+        instances:
+          title: Count of running instances
+          type: integer
+        capacity:
+          title: Total running capacity
+          type: integer
+      additionalProperties: false
+      required: [instances, capacity]
+    pending:
+      type: object
+      properties:
+        instances:
+          title: count of pending instances
+          type: integer
+        capacity:
+          title: Total pending capacity
+          type: integer
+      additionalProperties: false
+      required: [instances, capacity]
+    spotReq:
+      type: object
+      properties:
+        instances:
+          title: Count of open spot requests
+          type: integer
+        capacity:
+          title: Total capcity of open spot requests
+          type: integer
+      additionalProperties: false
+      required: [instances, capacity]
+    maxCapacity:
+      title: Maximum capacity of the workerType
+      type: integer
+  additionalProperties: false
+  required:
+    - running
+    - pending
+    - spotReq
+    - maxCapacity
```

Comment:
Is there any reason for structuring this like this rather than having:
`runningCapacity, runningInstances, pendingCapacity, pendingInstances, requestedSpotInstances, requestedSpotCapacity`

Nesting objects for structure is rarely desirable... I've long regretted `task.metadata`, the members should just have been top-level properties. (just figured I would warn you).